aggregation/aggregation_rule.go

```diff
@@ -36,6 +36,15 @@ func NewAggregationRule(aggSpec models.AggregationSpecification) Rule {
 	if aggSpec.Function == "" {
 		log.Fatalf("Rule %s must have a function", aggSpec.Name)
 	}
+outer:
+	for _, rKey := range aggSpec.Rollup.GroupedDimensions {
+		for _, key := range aggSpec.GroupedDimensions {
+			if key == rKey {
+				continue outer
```

Comment:
While this does work, I think adding InLIst() and using it would be a lot clearer